doorway-ui-components/src/headers/DoorwayHero.scss

```diff
@@ -68,6 +68,10 @@
   h1 {
     margin-bottom: var(--doorway-hero-between-margin);
   }
+
+  .field {
```

Comment:
Since this color contrast issue is dependent on the bg-color, I felt changing it in this way would help to contain the updates and leave other fields unaffected/our ui-c more aligned.

Reply:
I'm wondering if `.field` isn't the best class to target. Looks like DoorwayHero is only used on the home page, so could you just place it directly in `.doorway-hero_inner`?

Reply:
@ludtkemorgan Do you mind expanding? I don't think I'm following what "it" is in this case. As for the field class, I could initialize the variable within the field-note class if that would be preferred but initially was following pattern on where the other variables were placed.

Reply:
This might just be a personal preference, but I'm not a huge fan of targeting classes that we don't control. In this case `.field` comes from the form components. So to override the `--field-note-font-color` specifically on the DoorwayHero rather than using .field we could just put under the .doorway-hero-inner directly (just delete lines 72 and 74). Unless that doesn't work because of specificity rules

This is all under the assumption that the change here was to target all field-notes in this hero only. So if not, then you can ignore my suggestion